tests/rule_based_profiler/bobster_user_workflow_fixture.py

```diff
@@ -0,0 +1,58 @@
+import pytest
+
+# TODO: Move these fixtures to integration tests
+from great_expectations.data_context.util import file_relative_path
+
+
+@pytest.fixture
+def bobster_columnar_table_multi_batch_normal_mean_5000_stdev_1000():
+    """
+    # TODO: <Alex>ALEX -- Add DocString</Alex>
```

Comment:
Yeah, <Alex>Alex should do that</Alex>

Reply:
@jcampbell I will follow up on all remaining DocStrings once we gather all feedback for this PR.  In the specific case of `Bobster`, we generate 36 months' worth of data as CSV files, where the number of rows follows a Normal distribution with the mean of 5,000 and the standard deviation of 1,000 -- this data set is used to test the "bootstrap" method.  Thanks.

Reply:
This has been done.